src/Distributions.jl

```diff
@@ -65,6 +65,7 @@ export
     MultivariateMixture,
 
     # distribution types
+    AffineDistribution,
```

Comment:
I suggest not exporting `AffineDistribution`. It is not intended to be used by users directly, so exporting it should not be necessary. Moreover, non-exported types can be changed without worrying about making breaking changes.
```suggestion
```